ingeniamotion/fsoe_master/handler.py

```diff
@@ -408,6 +429,9 @@ def set_pdo_maps_to_slave(self) -> None:
             rpdo_maps=[self.safety_master_pdu_map], tpdo_maps=[self.safety_slave_pdu_map]
         )
 
+        # Subscribe to process data events
+        self.subscribe_to_process_data_events()
```

Comment:
this subscription lifecycle does not make sense to me.
If you start/stop/start you are not subscribed to the events.
maybe the pdo maps are already set to the slave and you are not setting them again...